File: Travel Management/Controllers/Employee.js
function saveNewEmployee()
{
    var employeeobj = {};

    employeeobj.FirstName = $("#EmpFirstName").val();
    employeeobj.LastName = $("#EmpLastName").val();
    employeeobj.Email = $("#EmpEmail").val();
    employeeobj.Phone = $("#EmpPhone").val();
    employeeobj.Department = $("#EmpDepartment").val();
    employeeobj.DOJ = dateformateChange( $("#DOJ").val());
    employeeobj.Type = $("#designation").val();

    $.ajax({
        type: "POST",
        url: "WebService.asmx/saveNewEmployee",
        data: '{employee: ' + JSON.stringify(employeeobj) + '}',
        contentType: "application/json",
        dataType: "json",
        success: function (response) {
            toastr.success("New Employee Created Successfully !", {
                showMethod: "slideDown",
                hideMethod: "slideUp",
                timeOut: 2e3
            });
            clearform();
        },
        error: function (response) {
            toastr.warning("Problem While Creating New Employee !", {
                showMethod: "slideDown",
                hideMethod: "slideUp",
                timeOut: 2e3
            });
        }

    });
}

function clearform()
{
   $("#EmpFirstName").val('');
   $("#EmpLastName").val('');
   $("#EmpEmail").val('');
   $("#EmpPhone").val('');
   $("#EmpDepartment").val('');
   $("#DOJ").val('');
   $("#Designation").val('');
}


function dateformateChange(date) {
    var datepart = date.split("/");
    return datepart[2] + "-" + datepart[1] + "-" + datepart[0];
}

File: Travel Management/BusinessEntities/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Travel_Management.BusinessEntities
{
    public class Employee
    {
        public string id { get; set; }
        public String EmpId { get; set; }

        public String FirstName { get; set; }
        public String LastName { get; set; }

        public String Email { get; set; }
        public String Password { get; set; }

        public String Phone { get; set; }
        public String Department { get; set; }
        public String Type { get; set; }
        public String DOJ { get; set; }

        public String ApproverManagerId { get; set; }

        public String OTP { get; set; }
    }
}